packages/studio-be/src/core/bots/bot-service.ts

```diff
@@ -63,7 +63,13 @@ const BotCreationSchema = Joi.object().keys({
       id: Joi.string()
     }
   },
-  locked: Joi.bool()
+  locked: Joi.bool(),
+  isCloudBot: Joi.bool().optional(),
+  cloud: Joi.object({
+    oauthUrl: Joi.string().optional(),
```

Comment:
Check that it's a valid URL?

Reply:
Can we do that with joi? If not, it'll require some more checks after the joi validation. 

Isn't it slightly overkill? Do you agree?

Reply:
@laurentlp url validation here may be overkill given that it's not the user giving us the url. It will be stored in our code base.

https://github.com/botpress/botpress/blob/70bee0c55d2a1d015c51bb57671d977f77bd30c2/packages/bp/src/admin/ui/src/workspace/bots/CreateBotModal.tsx#L132-L136